src/Core/Content/Cookie/Struct/CookieStruct.php

```diff
@@ -0,0 +1,100 @@
+<?php declare(strict_types=1);
+
+namespace Shopware\Core\Content\Cookie\Struct;
+
+use Shopware\Core\Framework\Log\Package;
+use Shopware\Core\Framework\Struct\Struct;
+
+#[Package('framework')]
+class CookieStruct extends Struct
```

Comment:
```suggestion
abstract class CookieStruct extends Struct
```

you should not use it on its own right? it is either a CookieGroup or a CookieEntry, correct?

Reply:
Used in CookieService as a base to create the others. So abstract would only work if I create the others in a different way 🤔